docs_outdoornav_user_manual/features/foxglove_visualization.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Foxglove Visualization
+sidebar_label: Foxglove Visualization
+sidebar_position: 6
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+We have enabled Foxglove visualization as part of OutdoornNav, for improved debugging and data visualization and analysis.
+To access the Foxglove visualization interface, navigate to the following [URL](http://192.168.131.1/_/foxglove)
```

Comment:
Were we intending to link to 192.168.131.1 here?

Reply:
Yes, i think for the majority of sales we will be targeting the default base station setup where we access the robot @ 192.168.131.1. For husky observer i can probably specify that it will be at a different IP.

Reply:
i've added the change. are the web elements accesible at over the 132.xxx subnet when connected to the base station?

Reply:
Can you clarify what the "web elements" are and what IP/port they are on with the 192.168.131.x subnet?